### PR TITLE
PR: Add daemoniker

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by goanpeca** [comment]
Pinging @nerohmot

File: recipes/daemoniker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - daemoniker
```
Comment (marcelotrevisani) [inline recipes/daemoniker/meta.yaml:27]: can we try to add ``pip check`` here please?
```suggestion
    - daemoniker
  requires:
    - pip
  command:
    - pip check
```

File: recipes/daemoniker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+about:
+  home: https://github.com/Muterra/py_daemoniker
+  license: Unlicense
+  license_family: PUBLIC-DOMAIN
+  license_file: UNLICENSE.txt
```
Comment (marcelotrevisani) [inline recipes/daemoniker/meta.yaml:33]: I believe the license here is not correct, could you please double check?
also the license is being packaged on pypi so you do not need the license file
```suggestion
  license: LGPL-2.1
  license_family: LGPL
  license_file: LICENSE.txt
```

Comment (marcelotrevisani) [reply recipes/daemoniker/meta.yaml:33]: the ``LGPL-2.1`` is the correct license for the version ``0.2.3``
on the main branch they changed, but we cannot distribute the ``0.2.3`` as the latest license I believe

File: recipes/daemoniker/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  description: |
+    Daemoniker also provides several utility tools for the resulting daemons.
+    In particular, it includes cross-platform signaling capability for the created daemons.
+  doc_url: https://daemoniker.readthedocs.io/en/latest/
+  dev_url: https://github.com/Muterra/py_daemoniker
+
+extra:
+  recipe-maintainers:
+    - goanpeca
+    - nerohmot
```
Comment (marcelotrevisani) [inline recipes/daemoniker/meta.yaml:44]: Hi @nerohmot , can we have a confirmation from you that you would like to maintain this recipe as well please?

File: recipes/daemoniker/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/daemoniker/meta.yaml:33]: when they release a new version you can change the license to unlicensed

File: recipes/daemoniker/meta.yaml (hunk 1 above)
Comment (goanpeca) [reply recipes/daemoniker/meta.yaml:27]: Fixed thanks :)

File: recipes/daemoniker/meta.yaml (hunk 2 above)
Comment (goanpeca) [reply recipes/daemoniker/meta.yaml:33]: You are right, thanks!

File: recipes/daemoniker/meta.yaml (hunk 3 above)
Comment (goanpeca) [reply recipes/daemoniker/meta.yaml]: Actually no need to co-maintain this @nerohmot, I can handle it :)

**Comment by goanpeca** [comment]
This one is ready :)

**Review by marcelotrevisani — APPROVED** [review]

**Comment by goanpeca** [comment]
Thanks ! 🥳

**Pull request merged**